### ISSUE TITLE
Consider reinstating env_package

### ISSUE THREAD

**Issue by turbomam** [GenomicsStandardsConsortium/mixs#387]
I understand that `env_package` was removed in MIxS 6. I think we discussed this some in person since I've been coming to the MIxS meetings. I have reread issues #64 and  #87. I still don't understand the reasoning, and there's a general consensus among NMDC developers that `env_package` is in important **data storage** field. (Although I can see that it might not be exposed to the users doing the data entry.)

From #64 

> it's not really a field the user can manually adjust, its implicit from the package they select and will be used by whatever tools we build for validation so that we know which checklist package to validate against.

Is there some explar tool now that implements validation of data values based on a previously selected  environmental package? The NMDC DataHarmonizer does ask users to select an environmental package before entering the biosample metadata. We are saving that selection in a slot called `env_package` and would love for that terms to remain defined in MIxS!

CC @mslarae13

**Comment by only1chunts** [comment]
Hi, nice explanation, thanks! I hadn't considered the behind the scenes mechanisms here, my thoughts were purely from a front end users point of view, i.e. they select a package and use it, so dont want to then write in a field saying which env package was used, its just implicit. 
With regards to validation tools or anything else, actually does it really matter which package was used? The only validation in terms of mandatory or not are based on the checklist (not the package), and any validation of the individual terms are based on the terms not the env-package, so I dont think there is any need to know which package was used in the creation step is there?

In short, my question is - What tools would require the knowledge of the env_package being used at the creation step?

**Comment by ramonawalls** [comment]
This is being handled elsewhere. We do not want to reinstate the term env_package

**Comment by turbomam** [comment]
Where is the elsewhere that it is being handled?
